plugins/CoreVisualizations/templates/_dataTableViz_htmlTable_comparisons.twig

```diff
@@ -0,0 +1,53 @@
+{% set lastSeenComparePeriod = false %}
+{%- for rowId, row in dataTable.getRows() -%}
+    {% set comparePeriod = row.getMetadata('comparePeriodPretty') %}
+    {% if lastSeenComparePeriod != comparePeriod %}
+    <tr class="comparePeriod">
+        <td colspan="{{ properties.columns_to_display|length }}">
+            {{ comparePeriod }}
+        </td>
+    </tr>
+    {% endif %}
+    {% set overrideParams = {} %}
+
+    {% set compareSegmentIndex = comparisonSegmentIndices[row.getMetadata('compareSegment')|default('')] %}
+    {% set comparePeriodIndex = comparePeriodIndices[row.getMetadata('comparePeriod')|default('')][row.getMetadata('compareDate')|default('')] %}
+
+    {% if row.getMetadata('compareSegment')|default is not empty %}{% set overrideParams = overrideParams|merge({ segment: row.getMetadata('compareSegment'), compareSegments: '' }) %}{% endif %}
+    {% if row.getMetadata('comparePeriod')|default is not empty %}{% set overrideParams = overrideParams|merge({ period: row.getMetadata('comparePeriod'), comparePeriods: '' }) %}{% endif %}
+    {% if row.getMetadata('compareDate')|default is not empty %}{% set overrideParams = overrideParams|merge({ date: row.getMetadata('compareDate'), compareDates: '' }) %}{% endif %}
+    {% set idSubtable = row.getMetadata('idsubdatatable_in_db') %}
+    <tr
+        {% if idSubtable != false %}data-idsubtable="{{ idSubtable|e('html_attr') }}"{% endif %}
+        data-comparison-segment="{{ compareSegmentIndex }}"
```

Comment:
I suppose we need to add escapes like `|e('html_attr')`? But I suppose it's a number here at the moment so might not be needed